src/helpers/date.js

```diff
@@ -19,3 +30,47 @@ export const formatTime = (time, format = "h:mm a") => {
 export const dateFromObjectId = (id) => {
     return dayjs(new Date(Number.parseInt(id.slice(0, 8), 16) * 1000));
 };
+
+export const now = (date, unit) => {
+    if (!date) {
+        if (unit) {
+            return dayjs.tz().startOf(unit);
+        }
+
+        return dayjs.tz();
```

Comment:
```suggestion
        return unit ? dayjs.tz().startOf(unit); : dayjs.tz();
```

You can update my seller app PR too